config/database.yml

```diff
@@ -1,19 +1,15 @@
 # TODO: Determine production environment before configuring db
 
 development:
-  adapter: postgresql
+  adapter: sqlite3
```

Comment:
but we all need to run a postgresql server in order to get anything to work which I feel is too much trouble